src/services/uniswap/UniswapPriceService.ts

```diff
@@ -1,19 +1,19 @@
 import {inject, injectable} from 'inversify';
 
 import {UniswapPoolPrice} from './UniswapPriceScanner';
-import {MongoDBPriceRepository} from '../../repositories/MongoDBPriceRepository';
+import {PriceRepository} from '../../repositories/PriceRepository';
 
 @injectable()
 export class UniswapPriceService {
-  @inject(MongoDBPriceRepository) priceRepository!: MongoDBPriceRepository;
+  @inject(PriceRepository) priceRepository!: PriceRepository;
 
-  private readonly SOURCE = 'uniswapv3';
+  static readonly SOURCE = 'uniswapv3';
 
   async savePrices(prices: UniswapPoolPrice[]): Promise<void> {
     await this.priceRepository.saveBatch(
       prices.map((p) => ({
-        source: this.SOURCE,
-        symbol: p.symbol,
+        source: UniswapPriceService.SOURCE,
+        symbol: p.symbol.toUpperCase(),
```

Comment:
why we need to change symbol and use uppercase?

Reply:
the symbol is lowercase from the pool(eg:usdc~blank) and it would be better to save it in the format that we should consume when creating the leaves, also I think that I should use replace("~", "-") here as well